R/parsing_partitions.R

```diff
@@ -0,0 +1,62 @@
+setwd('/home/striker/Desktop/UCLA/BS Research Paper/SideWork')
+
+#As a start, try turning the bit that parses the .partitions file into a function 
+#that returns a data frame with three columns, 
+
+#the first being the gene name, 
+#the second being the starting offset, 
+#and the third being the ending offset.
+
+parsing_partitions <- function(file_name) {
+  #Need to return a dataframe with 3 columns:
+  #1st column: The gene name
+  gene_names <- c()
+  #2nd column: The starting offset
+  starting_offset <- c()
+  #3rd column: The ending offset
+  ending_offset <- c()
+  
+  
+  fileName <- file_name
+  partions_strings <- readChar(fileName, file.info(fileName)$size)
+  
+  partion_strings_newline_split <- scan(text = partions_strings, what = "\n")
```

Comment:
I'm a little confused by this line since the `what = ` parameter to `scan` should indicate the type of the data.